src/data/java/net/dries007/tfc/data/providers/BuiltinRecipes.java

```diff
@@ -268,9 +268,6 @@ public void accept(ResourceLocation id, Recipe<?> recipe, @Nullable AdvancementH
                 blocks.get(SoilBlockType.FARMLAND).get(),
                 blocks.get(SoilBlockType.ROOTED_DIRT).get()
             ), blocks.get(SoilBlockType.DIRT).get().defaultBlockState()));
-            add(new LandslideRecipe(BlockIngredient.of(
```

Comment:
It looks like the replacement tags for this (and cracked mud/earth) are still missing. If you need the location, in `BuiltinBlockTags`, look for `tag(CAN_LANDSLIDE)` and add the following:

```java
    .add(TFCBlocks.SOIL.get(SoilBlockType.MUD))
    .add(TFCBlocks.SOIL.get(SoilBlockType.CRACKED_EARTH))
    .add(TFCBlocks.SOIL.get(SoilBlockType.SALTED_EARTH))
```

Reply:
Ah yes, I missed that when moving things around. Fixed now